.travis.yml

```diff
@@ -15,6 +15,7 @@ script:
       rm bears/Constants.py;  # There are no tests covering this module
       rm bears/c_languages/CSharpLintBear.py tests/c_languages/CSharpLintBearTest.py;
       rm bears/java/InferBear.py tests/java/InferBearTest.py;
+      rm bears/haskell/GhcModBear.py tests/haskell/GhcModBearTest.py;
```

Comment:
yea, this removes the bear and its test, during the CI run.
We could improve that slightly by actually removing the bear and its test from the Docker image.